src/XMakeBuildEngine/Evaluation/Evaluator.cs

```diff
@@ -987,6 +987,42 @@ private void PerformDepthFirstPass(ProjectRootElement currentProjectOrImport)
                 DebuggerManager.BakeStates(Path.GetFileNameWithoutExtension(currentProjectOrImport.FullPath));
             }
 #endif
+            IList<ProjectImportElement> implicitImports = new List<ProjectImportElement>();
+
+            if (!String.IsNullOrWhiteSpace(currentProjectOrImport.Sdk))
+            {
+                // SDK imports are added implicitly where they are evaluated at the top and bottom as if they are in the XML
+                //
+                foreach (string sdk in currentProjectOrImport.Sdk.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries)
```

Comment:
This splitting logic is an example of what could go in a new SDKProjectElement type, since most likely other parts of the codebase may want to parse the SDK string.